docs/changelog.md

```diff
@@ -30,6 +30,15 @@ permalink: /changelog/
 * **concept-storage**, **service-sync-logins**
   * 🆕 New API: `PlacesStorage#warmUp`, `SyncableLoginsStorage#warmUp` - allows consumers to ensure that underlying storage database connections are fully established.
 
+* **browser-engine-gecko**, **browser-engine-gecko-beta**, **browser-engine-gecko-nightly**
+  * Added `GeckoEngineView#getInputResult()` to return an EngineView.InputResult indicating how a MotionEvent was handled by an EngineView.
+* **browser-toolbar**
```

Comment:
nit: spaces between component changelog entries.

Reply:
Ohh, thanks!
I wanted to group all changes together because they're the result of the same PR.
Will separate them to respect the general structure, thanks!

Reply:
@Mugurell I think that's a fair idea! Maybe make the `browser-toolbar` and `feature-session` nested points in that case?